python/plugins/db_manager/db_plugins/plugin.py

```diff
@@ -1095,13 +1095,31 @@ def definition(self):
             txt += u" DEFAULT %s" % self.default2String()
         return txt
 
+    def getComment(self):
+        # Function returning the comment of a field if exists
+        tab = self.table()
+            #SQL Query checking if a comment exists for the field
+        sql_cpt = u"Select count(*) from pg_description pd, pg_class pc, pg_attribute pa where relname = '%s' and attname = '%s' and pa.attrelid = pc.oid and pd.objoid = pc.oid and pd.objsubid = pa.attnum" % (tab.name, self.name)
+            #SQL Query that return the comment of the field
+        sql = u"Select pd.description from pg_description pd, pg_class pc, pg_attribute pa where relname = '%s' and attname = '%s' and pa.attrelid = pc.oid and pd.objoid = pc.oid and pd.objsubid = pa.attnum" % (tab.name, self.name)
+        c = tab.database().connector._execute(None, sql_cpt)# Execute Check query
+        res = tab.database().connector._fetchone(c)[0] #Store result
+        if res == 1: 
+            # When a comment exists
+            c = tab.database().connector._execute(None, sql) #Execute query
+            res = tab.database().connector._fetchone(c)[0] #Store result
+            tab.database().connector._close_cursor(c) #Close cursor
+            return res #Return comment
```

Comment:
```suggestion
            return res
```

Reply:
I wouldn't be as picky on these comments, a bit more than we do usually won't hurt ;)

Reply:
@m-kuhn I totally agree, we usually fall a bit short with comments, but in this PR the S/N ratio seemed a little too low for me.

Reply:
I agree that `return '' # Return void string` falls into the stating the obvious category, but that "res" is a "comment" is not that straightforward (it could also be fixed by different variable naming probably, but let's not exaggerate)